test/parallel/test-child-process-promisified.js

```diff
@@ -32,3 +32,22 @@ const execFile = promisify(child_process.execFile);
     assert(err.message.includes('doesntexist'));
   }));
 }
+const failingCodeWithStdoutErr =
+  'console.log("out");console.error("err");process.exit(1)';
```

Comment:
Some people have figured out that you can use template literals for getting good multiline JS code. :)